src/app/patients/results/observation-selector.html

```diff
@@ -1,73 +1,163 @@
 
-
+<link href="Assets/StyleSheets/observation-selector.css" rel="stylesheet" />
```

Comment:
This is not a valid location for a link tag. Also styles should go into src/sass folder, not to some random place. If you want to leave them here, you'll have to modify the build process to include in the output, though I do not recommend this way.